src/rngs/jitter.rs

```diff
@@ -47,10 +47,90 @@ const MEMORY_SIZE: usize = MEMORY_BLOCKS * MEMORY_BLOCKSIZE;
 /// Use of `JitterRng` is recommended for initializing cryptographic PRNGs when
 /// [`OsRng`] is not available.
 ///
+/// `JitterRng` can be used without the standard library, but not conveniently.
+/// You have to provide a high-precision timer, and have to carefully follow the
+/// instructions of [`new_with_timer`].
+///
 /// This implementation is based on
 /// [Jitterentropy](http://www.chronox.de/jent.html) version 2.1.0.
 ///
-/// [`OsRng`]: ../os/struct.OsRng.html
+/// # Quality testing
+///
+/// [`JitterRng::new()`] automatically does its own quality testing. But before
+/// using `JitterRng` on untested hardware, after changes that could effect how
+/// the code is optimized, and after major compiler compiler changes, like a new
+/// LLVM version, it is recommend to run the much more stringent
+/// [NIST SP 800-90B Entropy Estimation Suite](
+/// https://github.com/usnistgov/SP800-90B_EntropyAssessment).
+///
+/// Use the following code using [`timer_stats`] to collect the data:
```

Comment:
Including the quality-testing warning in this documentation is fine, but the code and instructions is a bit much IMO. Have you considered putting this program in `examples/` to make it simpler to run?

Reply:
I think that is a good idea. Sounds like a bit more work than I'd like to put in it at the moment t.b.h. A few things I don't yet have the answer to: Do examples have a way to not run them when testing? Where to keep the instructions, in the `JitterRng` documentation in in the source of the 'example'? How do we make clear to users that this is not like the other examples (to not scare them away :smile:)? Maybe by adding some readme to the examples dir?